[Console] Add image support to `QuestionHelper` and `#[Ask]` (#63293)
symfony/symfony: src/Symfony/Component/Console/CHANGELOG.md

---

 * [BC BREAK] Add `object` support to input options and arguments' default by changing the `$default` type to `mixed` in `InputArgument`, `InputOption`, `#[Argument]` and `#[Option]`

>  * Add image paste support for `#[Ask]` with `InputFile` or `SplFileInfo` types, supporting Kitty Graphics and iTerm2 protocols

Reviewer: ```suggestion
 * Support pasting images with `#[Ask]` on `InputFile` or `SplFileInfo` types, supporting Kitty Graphics and iTerm2 protocols
```
